selftests/functional/test_basic.py

```diff
@@ -1137,18 +1121,6 @@ def test_json_plugin_errortest(self):
         self.run_and_check('errortest.py', exit_codes.AVOCADO_TESTS_FAIL,
                            1, 1, 0, 0)
 
-    @unittest.skipIf(not ECHO_BINARY, 'echo binary not available')
-    def test_ugly_echo_cmd(self):
-        data = self.run_and_check('"/bin/echo -ne foo\\\\\\n\\\'\\\\\\"\\\\\\'
```

Comment:
I'd prefer keeping this test, because IIRC it was based on Avocado not being able to even execute this command. Please use `--external-runner /bin/echo '-ne foo\\\\\\n\\\'\\\\\\"\\\\\\...` to run this test.